### PR TITLE
Fix segfault that occurs when mappedOps does not contain op during unpatch

### PR DESCRIPTION
**Pull request by typerSniper**

### CODE REVIEW

File: src/quartz/tasograph/tasograph.cpp [hunk 1]
```diff
@@ -2398,6 +2398,8 @@ bool Graph::xfer_appliable(GraphXfer *xfer, Op op) const {
   if (!success)
     // If failed, the unmatch is already done in _pattern_matching.
     return false;
+
+  success = _loop_check_after_matching(xfer);
```
Comment (xumingkuan) [inline src/quartz/tasograph/tasograph.cpp:2402]: The variable `success` seems unused here to me.

**Comment by typerSniper** [comment]
close pr

**Comment by typerSniper** [comment]
delete pr

**Comment by xumingkuan** [comment]
I assume you wanted to close this PR here.

**Pull request closed without merging**